samples/HubConnectionSample/HubConnectionSample.cpp

```diff
@@ -6,23 +6,24 @@
 #include "hub_connection.h"
 #include "log_writer.h"
 #include <future>
+#include "signalr_value.h"
 
 class logger : public signalr::log_writer
 {
     // Inherited via log_writer
     virtual void __cdecl write(const std::string & entry) override
     {
-        //std::cout << utility::conversions::to_utf8string(entry) << std::endl;
+        std::cout << entry << std::endl;
     }
 };
 
 void send_message(signalr::hub_connection& connection, const std::string& message)
 {
-    web::json::value args{};
-    args[0] = web::json::value(utility::conversions::to_string_t(message));
+    std::vector<signalr::value> arr { std::string("c++"), message };
```

Comment:
What is the string "c++" used for?

Reply:
I am currently running this sample against the ChatSample in the AspNetCore repo, and that requires a "name" for the first param.